.github/workflows/dart_ci.yml

```
@@ -0,0 +1,59 @@
+name: Dart CI
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'test_consume_*'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [ 2.13.4, stable, beta, dev ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dart-lang/setup-dart@v0.2
+        with:
+          sdk: ${{ matrix.sdk }}
+
+      - id: install
+        name: Install dependencies
+        run: dart pub get
+
+      - name: Validate dependencies
+        run: dart run dependency_validator
+        if: always() && steps.install.outcome == 'success'
+
+      - name: Analyze project source
+        run: dart run dart_dev analyze
+        if: always() && steps.install.outcome == 'success'
+
+      - name: Run tests with ddc
+        run: dart run dart_dev test
+        if: always() && steps.install.outcome == 'success'
+
+      - name: Run tests with dart2js
+        run: dart run dart_dev test --release
+        if: always() && steps.install.outcome == 'success'
```

For now we should just not use dart_dev to run the tests.

```suggestion
      - name: Run tests with ddc
        run: dart run build_runner test
        if: always() && steps.install.outcome == 'success'

      - name: Run tests with dart2js
        run: dart run build_runner test --release
        if: always() && steps.install.outcome == 'success'
```